src/NHSD.BuyingCatalogue.Solutions.API/Controllers/AdditionalServiceController.cs

```diff
@@ -12,35 +12,35 @@ public sealed class AdditionalServiceController : ControllerBase
     {
         [HttpGet]
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "Cannot be viewed on swagger if static")]
-        public ActionResult<AdditionalServiceResult> Get([FromQuery] string[] solutionIds)
+        public ActionResult<IEnumerable<AdditionalServiceResult>> Get([FromQuery] string[] solutionIds)
         {
             if (solutionIds is null)
                 return NotFound();
 
-            return new AdditionalServiceResult
+            var additionalServiceResults = new List<AdditionalServiceResult>();
+
+            foreach (var solutionId in solutionIds)
             {
-                Id = SetAdditionalServiceId(solutionIds),
-                Name = "Write on Time Additional Service 1",
-                Summary = "Addition to Write on Time",
-                Solution = new AdditionalServiceSolutionResult
+                additionalServiceResults.Add(new AdditionalServiceResult
                 {
-                    Id = "100000-001",
-                    Name = "Write on Time"
-                }
-            };
+                    AdditionalServiceId = SetAdditionalServiceId(solutionId),
+                    Name = "Write on Time Additional Service 1",
+                    Summary = "Addition to Write on Time",
+                    Solution = new AdditionalServiceSolutionResult
+                    {
+                        SolutionId = "100000-001",
+                        Name = "Write on Time"
+                    }
+                });
+            }
+
+            return additionalServiceResults;
         }
 
-        private static string SetAdditionalServiceId(string[] solutionIds)
+        private static string SetAdditionalServiceId(string solutionId)
         {
             var additionalServiceId = string.Empty;
```

Comment:
This could arguably just be a single line now, and doubly-arguably doesn't even need the function, but it's ok as the canned data will be getting canned itself, soon.